sync-gsheet-csv.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+PROBE_MOBILE_STRINGS_SHEET="https://docs.google.com/spreadsheets/d/1GPkr_OyNhXJRTXnseyNbo2P4Du-i_W9kJdqjZG4HVkM/export?format=csv"
+
+CSV_FILE_PATH=probe-mobile/en/strings.csv
+
+echo -n "Fetching latest source strings from spreadsheet..."
+
+curl -sL $PROBE_MOBILE_STRINGS_SHEET -o $CSV_FILE_PATH
```

Comment:
```suggestion
curl -fsL $PROBE_MOBILE_STRINGS_SHEET -o $CSV_FILE_PATH
```

I think we want to add this flag to the command line:

```
       -f, --fail
              (HTTP) Fail silently (no output at all) on server errors. This is
              mostly done to enable scripts etc to better deal with failed
              attempts. In normal cases when an HTTP server fails to deliver a
              document, it returns an HTML document stating so (which often also
              describes why and more). This flag will prevent curl from
              outputting that and return error 22.

              This method is not fail-safe and there are occasions where non-
              successful response codes will slip through, especially when
              authentication is involved (response codes 401 and 407).

              Example:
               curl --fail https://example.com

              See also --fail-with-body.
```